config/routes.rb

```diff
@@ -1,8 +1,10 @@
 Rails.application.routes.draw do
+  get 'book_tour/create'
   scope "(:locale)", locale: /en|vi/ do
     root "tour#index"
     get "/signup", to: "users#new"
-    resources :tours, only: %i("index")
+    resources :tour, only: %i(" index show ")
```

Comment:
xóa nháy đi, mấy cái khác sửa tương tự
```suggestion
    resources :tour, only: %i(index show)
```